internal/workflow.go

```diff
@@ -1413,6 +1413,13 @@ func WithWorkflowTaskList(ctx Context, name string) Context {
 	return ctx1
 }
 
+// WithWorkflowTaskListApply returns a copy of context and changes workflow tasklist with apply function.
```

Comment:
nit: maybe a better name is `WithWorkflowTaskListMapper`

Reply:
same in other places

Reply:
fixed